docs/GUIDE_TO_KOGITO_DEVS.md

```diff
@@ -0,0 +1,190 @@
+COPY jobs-service-8.0.0-SNAPSHOT-runner.jar ${KOGITO_HOME}/bin/kogito-jobs-service-runner.jar
+```
+**Note**: Usually, latest artifacts are with version  `8.0.0-SNAPSHOT`
+**Note**: You can check the location of the artifact in the image from the [kogito-images](https://github.com/kiegroup/kogito-images) repository.
+
+Now you are going to build the image. Inside the above created directory, run:
+
+```shell-script
+$ podman build -t <prefered-tag-name>  -f jobs-service-Dockerfile  .
+```
+
+The above command will build your image with `quay.io/<your-namespace>/<image-name>` and can be seen in the output of `$ podman images`
+
+**Note**: We added the `-f` flag to specify the `Dockerfile` in case you want to build multiple images, you can then have different files and specify them with the `-f` flag.
+This image needs to be on a public container registry from which it can be pulled later.
+```shell-script
+$ podman push quay.io/<your-namespace>/<image-name>
+```
+
+## Build the Kogito Service Image
+
+### Base Images
+  * kogito-quarkus-ubi8
+  * kogito-quarkus-jvm-ubi8
+  * kogito-springboot-ubi8
+
+As an example for building the Image for your Kogito custom Service we’re going to build the example  `process-business-rules-quarkus` from [kogito-examples](https://github.com/kiegroup/kogito-examples) repository.
+
+```shell-script
+$ git clone https://github.com/kiegroup/kogito-examples
+cd kogito-examples/process-business-rules-quarkus
+mvn clean package
+```
+Now create a Dockerfile inside the `kogito-examples/process-business-rules-quarkus`. With the following content:
+
+```Dockerfile
+FROM quay.io/kiegroup/kogito-quarkus-jvm-ubi8:latest
+
+COPY target/*-runner.jar $KOGITO_HOME/bin    
+COPY target/lib $KOGITO_HOME/bin/lib
+```
+
+```shell-script
+$ podman build --tag quay.io/<yournamespace>/process-business-rules-quarkus:latest .
+```
+It’ll build your image with the kogito service running on it.
+
+Now let’s push the above built image to a public container registry.
+
+```shell-script
+$ podman push quay.io/<yournamespace>/process-business-rules-quarkus:latest
+```
+
+## Install the Operator
+
+First we would need to enable the OLM in our Minikube cluster. For that, just run:
+
+```shell-script
+$ minikube addons enable olm
+```
+
+To launch the OLM console locally, clone the [operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager) repository and from the root of the project run: `$ make run-console-local`.
+ 
+This will run the operatorhub console on http://localhost:9000 
+
+**Note**: You will need to have [`jq`](https://stedolan.github.io/jq/manual/) installed and 9000 port available on the system.
+
+Create a different namespace where `kogito-operator` and all the dependent operator(s) will run
+
+```shell-script
+$ kubectl create ns kogito
+```
+
+Now open your browser and visit the OLM console on https://localhost:9000. Select `Operators > OperatorHub` and search for `kogito`.  
```

Comment:
```suggestion
Now open your browser and visit the OLM console on http://localhost:9000. Select `Operators > OperatorHub` and search for `kogito`.  
```

http not https, otherwise it won't open.

I have followed all instructions up to here successfully, but I could not find or search for kogito. This is what I get:

> ### No OperatorHub Items Found
> Please check that the OperatorHub is running and that you have created a valid OperatorSource. For more information about OperatorHub, please click here.

![image](https://user-images.githubusercontent.com/380791/88911214-d4ab2980-d25d-11ea-998d-3b10211e86b0.png)

I see that I need to set an OperatorSource. Which one?

Reply:
I noticied it happens with sometimes, not really sure why. I'll check this out. Can you try the further steps with the alternate way to install the operator, Note: that installs the operator in `my-kogito-operator` namespace so use that namespace with rest of the commands (with `-n` flag).

Reply:
I believe that takes some time for the OLM to index all the manifests out there.

Reply:
The problem was because the readiness probe was failing for the `operator-catalog` pod causing none of the operator to be displayed.